Description: Adds the dependency on junit (no longer provided by plexus-container-default)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
--- a/pom.xml
+++ b/pom.xml
@@ -42,6 +42,12 @@
       <artifactId>plexus-container-default</artifactId>
       <version>1.0-beta-3.0.7</version>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <version>4.12</version>
+      <scope>test</scope>
+    </dependency>
   </dependencies>
 
   <scm>
